src/plugins/graph/models/graph-model.ts

```diff
@@ -48,6 +48,7 @@ export const GraphModel = TileContentModel
     // keys are AxisPlaces
     axes: types.map(AxisModelUnion),
     plotType: types.optional(types.enumeration([...PlotTypes]), "casePlot"),
+    hasXYDefaultAxisLabels: types.optional(types.boolean, false),
```

Comment:
I think this would be better as a setting available via `useSettingFromStores()` rather than as a model property, perhaps simply `defaultAxisLabels`. That way it can be configured on a unit or problem level rather than a graph-by-graph level.